docs/registry/_index.md

```diff
@@ -34,18 +34,21 @@ The [Viam registry](https://app.viam.com/registry) is the central place where yo
 **Modular resources** that add capabilities to your machine beyond what is built into `viam-server`
 
 {{% /manualcard %}}
-{{% manualcard link="#ml-models" %}}
+{{% manualcard link="/registry/ml-models/" %}}
 
 **ML models** to deploy with machine applications like computer vision
 
 {{% /manualcard %}}
-{{% manualcard link="#ml-training-scripts" %}}
+{{% manualcard link="/registry/training-scripts/" %}}
 
 **Training scripts** to train and produce ML models in the Viam cloud for custom machine learning
 
 {{% /manualcard %}}
 {{< /cards >}}
 
+The registry contains contributions from Viam as well as from external contributors.
+If you don't find what you need in the registry, you can create your own integrations, and you can choose to share them to the registry.
```

Comment:
I thought it might be important to explain where the registry contents are coming from, as part of what the registry even is.

Reply:
I find it awkward. I think the rest of this page should make it clear you can contribute but that should be who to convey that concept.

I think change the intro to:

The Viam registry is the storage and distribution system for:


Because it's not for browsing actually its a place to pull the software/models/scripts from